### PR TITLE
Add fields to DataOceanUser

### PR DESCRIPTION
**Pull request by PyDev777**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -39,6 +39,9 @@ def create_superuser(self, email, password, **extra_fields):
 class DataOceanUser(AbstractUser):
     username = None
     email = models.EmailField(_('email address'), unique=True)
+    organization = models.CharField(max_length=255, default='')
+    position = models.CharField(max_length=150, default='')
```
Comment (roman-tiukh) [inline users/models.py:43]: blank = True

Comment (roman-tiukh) [reply users/models.py:43]: what is position?

File: users/models.py [hunk 2]
```diff
@@ -39,6 +39,9 @@ def create_superuser(self, email, password, **extra_fields):
...
+    date_of_birth = models.DateField(null=True, blank=True)
```
Comment (roman-tiukh) [inline users/models.py:44]: default = None

File: users/models.py (hunk 1 above)
Comment (PyDev777) [reply users/models.py]: Fixed

File: users/models.py (hunk 2 above)
Comment (PyDev777) [reply users/models.py]: Fixed

**Pull request merged**